src/internal/components/dropdown/dropdown-fit-handler.ts

```diff
@@ -104,28 +115,34 @@ export const getInteriorAvailableSpace = ({
     ? AVAILABLE_SPACE_RESERVE_MOBILE_HORIZONTAL
     : AVAILABLE_SPACE_RESERVE_DEFAULT;
   const {
-    bottom: triggerBottom,
-    top: triggerTop,
-    left: triggerLeft,
-    right: triggerRight,
-  } = trigger.getBoundingClientRect();
+    insetBlockEnd: triggerBottom,
+    insetBlockStart: triggerTop,
+    insetInlineStart: triggerLeft,
+    insetInlineEnd: triggerRight,
+  } = getLogicalBoundingClientRect(trigger);
 
   return overflowParents.reduce(
-    ({ above, below, left, right }, overflowParent) => {
-      const currentAbove = triggerBottom - overflowParent.top - AVAILABLE_SPACE_RESERVE_VERTICAL;
-      const currentBelow = overflowParent.height - triggerTop + overflowParent.top - AVAILABLE_SPACE_RESERVE_VERTICAL;
-      const currentLeft = triggerLeft - overflowParent.left - AVAILABLE_SPACE_RESERVE_HORIZONTAL;
+    ({ blockStart, blockEnd, inlineStart, inlineEnd }, overflowParent) => {
+      const currentAbove = triggerBottom - overflowParent.insetBlockStart - AVAILABLE_SPACE_RESERVE_VERTICAL;
```

Comment:
Since we are refactoring internal APIs already, why not these variable names too to make it consistent?

Reply:
Updated!